src/game/createGameBoards.js

```diff
@@ -0,0 +1,85 @@
+import { SHIPS_LIST } from "./constants";
+import { getDataFromStorage } from "./../utils/localStorage/localStorage";
+
+const GAME_BOARD_WIDTH = 10;
+const gameBoard = document.querySelectorAll(".board__fields");
```

Comment:
`gameBoard`, `numberWrapper`, `shipsList`, `playerNameField`, `shipsHeading`, `buttons` - all of those could be named using `SCREAMING_SNAKE_CASE` you don't declare them in the single function body but out of it. Btw. you have a typo in `numbWrapper` 👇